WBooks/Repository/BookRepository.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BookRepository.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 23/08/21.
+//
+
+import Foundation
+import Alamofire
+
+internal class BookRepository {
+        
+    public func fetchBooks(onSuccess: @escaping ([Book]) -> Void, onError: @escaping (Error) -> Void) {
+
+        let url = URL(string: "https://ios-training-backend.herokuapp.com/api/v1/books")!
+
+        AF.request(url).responseJSON { response in
```

Comment:
We can use responseDecodable instead of responseJSON

Reply:
I will correct it

Reply:
Done